README.md

```diff
@@ -1,132 +1,14 @@
-    --tls-cert 'sebak.crt' \
-    --log-level debug \
-    --secret-seed SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM \
-    --validator GBWCMWDUZK67YNUZ44UPNVFYZRSCCS4OLE6ORWD4ZLI2MVGY4KJDPHMO,https://localhost:12346 \ --validator GCZG7MBKRSS6MJVZOALYBJB5C223FSZ43MDTPX2O4UGQTCXTHWBDNUB6,https://localhost:12347
-```
-
-To run multiple sebak nodes on the local machine, then you must set up the different db paths each:
-
-- node12345
-```
-$ sebak node \
-    --network-id 'this-is-test-sebak-network' \
-    --endpoint "https://localhost:12345" \
-    --tls-key 'sebak.key' \
-    --tls-cert 'sebak.crt' \
-    --log-level debug \
-    --storage=file:///tmp/db5 \
-    --secret-seed SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM \
-    --validator GDPQ2LBYP3RL3O675H2N5IEYM6PRJNUA5QFMKXIHGTKEB5KS5T3KHFA2,https://localhost:12346 \
-    --validator GCZG7MBKRSS6MJVZOALYBJB5C223FSZ43MDTPX2O4UGQTCXTHWBDNUB6,https://localhost:12347
-```
-
-- node12346
-```
-$ sebak node \
-    --network-id 'this-is-test-sebak-network' \
-    --endpoint "https://localhost:12346" \
-    --tls-key 'sebak.key' \
-    --tls-cert 'sebak.crt' \
-    --log-level debug \
-    --storage=file:///tmp/db6 \
-    --secret-seed SBGJDQ2J4PIYU7JVGKIBLNF6X3DOEVW3I4W2T77M2B47X2MPSUNXZ7T7 \
-    --validator GBNUTWSM4FRSEULVMHZF7NFQWIBGEDF5X5OHXFOZJB6SH5MIEDEJEJ2F,https://localhost:12345 \
-    --validator GCZG7MBKRSS6MJVZOALYBJB5C223FSZ43MDTPX2O4UGQTCXTHWBDNUB6,https://localhost:12347
-```
-
-- node12347
-```
-$ sebak node \
-    --network-id 'this-is-test-sebak-network' \
-    --endpoint "https://localhost:12347" \
-    --tls-key 'sebak.key' \
-    --tls-cert 'sebak.crt' \
-    --log-level debug \
-    --storage=file:///tmp/db7 \
-    --secret-seed SDQKKG2MBSAXVLUE5JFNM7MXQ7MV7WPRIEOS7U7KLWFDKYDKXTLSSRTC \
-    --validator GBNUTWSM4FRSEULVMHZF7NFQWIBGEDF5X5OHXFOZJB6SH5MIEDEJEJ2F,https://localhost:12345 \
-    --validator GDPQ2LBYP3RL3O675H2N5IEYM6PRJNUA5QFMKXIHGTKEB5KS5T3KHFA2,https://localhost:12346
-```
-
-## Spinning a test net using Docker
-
-To spawn a simple network, first build the docker image:
-```sh
-docker build . -t sebak
-# The build process creates a rather large image, which is regenarated every time
-# To clean up all orphaned images, you can run the following command
-docker rmi -f $(docker images -f "dangling=true" -q)
-```
-
-Then you can spawn 3 nodes using the following commands:
-```sh
-docker run --net host --rm -it --env-file=docker/node1.env sebak
-docker run --net host --rm -it --env-file=docker/node2.env sebak
-docker run --net host --rm -it --env-file=docker/node3.env sebak
-```
-
-Changing the number of test nodes should be fairly simple once #46 is solved.
+We welcome contributions by the community. Please open issues for any bug you find, or for enhancement requests. Pull requests are also very welcome. Before submitting, please read the [contribution guidelines](https://github.com/bosnet/sebak/blob/master/CONTRIBUTING.md).
```

Comment:
You can replace the link with just: `[contribution guidelines](CONTRIBUTING.md)`
https://blog.github.com/2013-01-31-relative-links-in-markup-files/